app/views/public/api_documentation.html.erb

```diff
@@ -409,6 +410,43 @@ curl -X POST -H "Token: secret-api-token" -H "Content-Type: application/json" -d
 
 <hr />
 
+<h4 id="sol-prices">Sol Prices</h4>
```

Comment:
Please use the same HTML format and the same wording for this whole block as for other API docs.